### DISCUSSION TITLE
新人入驻？火花骑士！

### DISCUSSION THREAD

**Discussion by XChi1231** [share121/inter-knot#100] [其他]
![1711981409909_format,jpg big](https://github.com/user-attachments/assets/6b642a40-5968-409f-bd76-bfc0ecc8671c)
西风骑士团火花骑士-可莉，前来报到！

**Comment by NOVAQUQ** [comment]
好耶！
![Uploading img_-1175557342_1690572464668.jpeg…]()

**Comment by qiu-1214** [comment]
可莉快跑，琴团长来了

**Reply by XChi1231** [reply]
求求你啦~求求你啦~帮帮可莉吧~
